packages/orbit-components/src/Popover/components/ContentWrapper.js

```diff
@@ -238,7 +250,7 @@ const PopoverContentWrapper = ({
 
   return (
     <>
-      <StyledOverlay shown={shown} isInsideModal={isInsideModal} />
+      <StyledOverlay shown={shown} isInsideModal={isInsideModal} ref={overlay} />
```

Comment:
Use `onMouseDown` on `StyledOverlay` instead of attaching and removing the event listener, that way we can test it. ✨

Reply:
thanks a lot for the test :) <3